starter-code/css/main.css

```diff
@@ -6,3 +6,176 @@ Green: #00B172
 White: #FFF
 
 */
+@import url('https://fonts.googleapis.com/css?family=Roboto:100,300,400,700');
+
+body {
+  margin: 0 auto;
+  font-family: 'Roboto', sans-serif;
+}
+
+.nav {
+  width: 100%;
+  background: white;
+  height: 7em;
+  position: fixed;
+  top: 0px;
+}
+
+.spotify-logo {
+  width: 11em;
+  float: left;
+  padding: 2em 8em;
+}
+
+.nav-links {
+  float: right;
+  padding: 3em 10em;
+}
+
+.nav-links > a {
+  text-decoration: none;
+  font-size: 20px;
+  padding: 0px 20px;
+  color: black;
+}
+
+#concert {
+  background-image: url('../images/landing.jpg');
+  background-size: cover;
+  width: 100%;
+  height: 60em;
+  background-repeat: no-repeat;
+}
+
+h1 {
+  color: white;
+  font-weight: 400;
+  padding-top: 400px;
+  text-align: center; 
+  font-size: 5em;
+  margin: 0;
+}
+
+.landingP {
+  color: white;
+  text-align: center;
+  width: 100%;
+  font-size: 1.75em;
+  font-weight: 200;
+}
+
+#container {
+  background: white; 
+}
+
+h3 {
+  font-size: 2.7em;
+  text-align: center;
+  font-weight: 500;
+  font-family: 'Roboto', sans-serif;
+  border-bottom: 5px #00B172 solid;
+  width: 8.2em;
+  margin: 4% 38%;
+}
+
+h4 {
+  color: #00B172;
+  font-size: 27px;
+  font-weight: 400;
+  text-align: center;
+}
+
+p {
+  font-size: 24px;
+  font-weight: 100;
+  text-align: center;
+}
+
+#spot {
+  width: 100%;
+  height: 35em;
+}
+
+#music {
+  float: left;
+  padding: 1em 5em 0em 15em;
+}
+
+#music > img {
+  width: 10em;
+  padding-left: 1.5em;
+}
+
+#HQ {
+  float: left;
+  padding: 2em 5em 0em 4em;
+}
+
+#HQ > img{
+  width: 10em;
+  padding-left: 2.5em;
+}
+
+#stream {
+  float: left;
+  padding: 1em 2em 0em 3em;
+}
+
+#stream > img{
+  width: 10em;
+  padding-left: 2em;
+}
+
+#yeezy {
+  clear: both;
+  background-color: #00B172;
+  width: 96%;
+  height: 58em;
+  margin: 2em;
+  background-image: url(../images/spotify-icon-white.png);
+  background-repeat: no-repeat;
+  background-size: 9em;
+  background-position: center;
+}
+
+.yeezyh4 {
+  font-size: 44px;
+  color: white;
+  font-weight: 600;
+  border-bottom: 4px white solid;
+  width: 12.6em;
+  margin: 2em 2em 0em 2em;
+}
+
+.kanyeh4 {
+  color: white;
+  font-weight: 600;
+  font-size: 30px;
+  text-align: left;
+  padding: .5em 0px 0px 3em;
+}
+
+#yeezy p {
+  font-size: 25px;
+  text-align: left;
+  color: white;
+  font-weight: 100;
+  padding-left: 3.5em;
+  line-height: 1.5;
+}
+
+#kanye {
+  float: left;
```

Comment:
You used the `float:left` property in a bunch of CSS classes ... When you start writing more CSS and you notice that you are repeating something over and over again, then its best to make it into its own class... So for example, you could have created a class called something like:
```css
.column-left {
  float: left;
}
```
and  then deleted `float:left` from all your classes in your HTML and then in your html you could have added `column-left` to the classes that would need a float left... for example:
```html
<div id="stream" class="column-left">
```
Or you could have made your life easier and decided to not use any #ids and just use classes instead, so then you could just have:
```html
<div class="column-left stream">
```
For now, Stick with CSS classes. They are reusable and their specificity can be easily reused.

Styling IDs goes against DRY principles and is something that should be avoided, if it makes sense. 


There are situations where you need to style a single object and you know it will never be shared with anything else in those rare case we will use ID, as long as you're aware of the maintainability risk and it's acceptable do what you need to do to get the job done. 

But in a situation like this , especially when learning , I think we should just not use it , and you used them WAY WAY to much in your HTML.